tests/composite_pk/models/tenant.py

```diff
@@ -53,7 +53,8 @@ class Post(models.Model):
 
 
 class TimeStamped(models.Model):
-    pk = models.CompositePrimaryKey("id", "created")
+    pk = models.CompositePrimaryKey("id", "created", "tenant_id")
     id = models.SmallIntegerField(unique=True)
     created = models.DateTimeField(auto_now_add=True)
+    tenant = models.ForeignKey(Tenant, on_delete=models.CASCADE, default=1)
```

Comment:
This seemed to be the least impactful place to adjust a model to have 3 composing fields, but I'm also okay with a new model.